src/GNDStk/Tree/test/write.test.cpp

```diff
@@ -231,43 +219,45 @@ SCENARIO("Testing GNDStk tree write() and operator<<") {
       // write *files*.
 
       // format::tree
-      WHEN("We write() or << the empty tree in format::tree format") {
+      WHEN("We write() the empty tree using format::tree") {
+         // write()
          {
```

Comment:
Could make these blocks using `THEN` to help clarify what is being done.

Reply:
Sure, I can go in and do that. The `THEN` messages throughout this file will be somewhat repetitive, basically saying something along the lines of "then we'll get the empty string" or "we'll get a declaration-node-only XML" or similar verbiage, but I can certainly do that.